test/add-returninfo-csv-parser.spec.js

```diff
@@ -0,0 +1,217 @@
+import test from 'tape'
+import AddReturnInfoCsvParser from 'add-return-info-csv-parser'
+import StreamTest from 'streamtest'
+import path from 'path'
+import { SphereClient } from 'sphere-node-sdk'
+import sinon from 'sinon'
+import fs from 'fs'
+import CONSTANTS from '../src/constants'
+
+let PROJECT_KEY
+
+if (process.env.CI === 'true')
+  PROJECT_KEY = process.env.SPHERE_PROJECT_KEY
+else
+  PROJECT_KEY = process.env.npm_config_projectkey
+
+const logger = {
+  error: console.error,
```

Comment:
Linter warnings here, `process.xxx` would be a solution but since the tests do not use it at all I would say put in noop functions.